Server/src/controllers/authenticationController.ts

```diff
@@ -38,17 +38,16 @@ export class AuthenticationController {
             return response.status(400).json("Request is invalid. Email or Password attribute missing")
         }
         else {
-            let requestParams: any = { ...request.query };
+            let requestParams: any = { ...request.body };
             let loginInfo: ILoginInformation = requestParams;
             let res: any = await this.callServiceForLogin(loginInfo, response, next);
             return res;
         }
     }
 
     private validateSignUpRequest(request: Request): boolean {
-
-        if (request.query.hasOwnProperty('email') && request.query.hasOwnProperty('password') && request.query.hasOwnProperty('firstName')
-            && request.query.hasOwnProperty('lastName') && request.query.hasOwnProperty('organizationName')) {
+        if (request.body.hasOwnProperty('email') && request.body.hasOwnProperty('password') && request.body.hasOwnProperty('firstName')
```

Comment:
If request.query is changed to request.body then this must be changes in the test files also

Reply:
tests have been fixed and adapted to the change